src/Site.js

```diff
@@ -385,6 +412,193 @@ Site.prototype.createPage = function (config) {
   });
 };
 
+/**
+ * Converts an existing Github wiki or docs folder to a MarkBind website.
+ */
+Site.prototype.convert = function (rootPath) {
+  return new Promise((resolve, reject) => {
+    this.addIndexPage(rootPath)
```

Comment:
Do `return this.addIndexPage(rootPath)...` directly.

See: [What is the explicit promise construction antipattern and how do I avoid it?](https://stackoverflow.com/q/23803743/8601760)

Reply:
Thanks for the link! I'll make the changes.